content/en/docs/v3.5/learning/why.md

```diff
@@ -118,13 +118,13 @@ Note that in the case of etcd keys, it can be locked efficiently because of the
 [etcd-linread]: ../api_guarantees/#isolation-level-and-consistency-of-replicas
 [etcd-mvcc]: ../data_model/
 [etcd-rbac]: ../../op-guide/authentication/#working-with-roles
-[etcd-recipe]: https://godoc.org/github.com/etcd-io/etcd/client/v3/experimental/recipes
+[etcd-recipe]: https://github.com/etcd-io/etcd/tree/main/client/v3/experimental/recipes
```

Comment:
All `godoc.org` links should be replaced by corresponding https://pkg.go.dev/go.etcd.io links, not links into GH.

It is possible that some of these entries don't exist anymore.